docs/development/updating_ossec.rst

```diff
@@ -82,6 +82,98 @@ a failing test which you then can make pass with a patch to the OSSEC rules:
 .. _syscheck: https://ossec-docs.readthedocs.io/en/latest/docs/manual/syscheck/index.html
 .. _2134: https://github.com/freedomofpress/securedrop/issues/2134
 
+
+How to add a new OSSEC rule?
+=============================
+
+There are two main files involved in this.
+
+- `install_files/securedrop-ossec-server/var/ossec/rules/local_rules.xml` the rules file
+- `install_files/securedrop-ossec-server/var/ossec/etc/local_decoder.xml` the decoder file
+
+
+The decoder file
+-----------------
+
+::
+
+    <!--
+      The default fwupd tries to auto-update and generates error.
+    -->
+    <decoder name="fwupd">
+      <program_name>fwupd</program_name>
+    </decoder>
+
+In the above example, we are creating a new `decoder` based on the
+`program_name` value. We can find this `program_name` value using the
+`/var/ossec/bin/ossec-logtest` command, you can paste the login as input to
```

Comment:
See the section above (L26, there may be some duplicate information here. Perhaps folding it in or maintaining a reference to it makes sense here.